apps/velo-external-db/test/e2e/app_data.e2e.spec.ts

```diff
@@ -49,106 +50,95 @@ describe(`Velo External DB Data REST API: ${currentDbImplementationName()}`,  ()
-            expect.toIncludeSameMembers([{ item: ctx.item }, data.pagingMetadata(1, 1)]))
+        await expect(data.queryCollectionAsArray(ctx.collectionName, [], undefined, authOwner, filterByDate)).resolves.toEqual({
+            items: expect.toIncludeSameMembers([ctx.item]),
+            pagingMetadata: data.pagingMetadata(1, 1)
+        })
     })
     
     testIfSupportedOperationsIncludes(supportedOperations, [ Projection ])('find api with projection', async() => {
         await schema.givenCollection(ctx.collectionName, [ctx.column], authOwner)
         await data.givenItems([ctx.item ], ctx.collectionName, authAdmin)
        
-        await expect(data.queryCollectionAsArray(ctx.collectionName, [], [ctx.column.name], authOwner)).resolves.toEqual(
-            expect.toIncludeSameMembers([{ item: { [ctx.column.name]: ctx.item[ctx.column.name], _id: ctx.item._id } }, data.pagingMetadata(1, 1)])
-        )                
+        await expect(data.queryCollectionAsArray(ctx.collectionName, [], [ctx.column.name], authOwner)).resolves.toEqual({
+            items: expect.toIncludeSameMembers([{ [ctx.column.name]: ctx.item[ctx.column.name], _id: ctx.item._id } ]),
+            pagingMetadata: data.pagingMetadata(1, 1)
+        })                
     })
     
     //todo: create another test without sort for these implementations
 
     test('insert api', async() => {
         await schema.givenCollection(ctx.collectionName, [ctx.column], authOwner)
 
-        const response = await axiosInstance.post('/data/insert', data.insertRequest(ctx.collectionName, ctx.items, false),  { responseType: 'stream', ...authAdmin })
+        const response = await axiosInstance.post('/data/insert', data.insertRequest(ctx.collectionName, ctx.items), { ...authAdmin })
 
-        const expectedItems = ctx.items.map(item => ({ item }))
-
-        await expect(streamToArray(response.data)).resolves.toEqual(expectedItems)
-        await expect(data.queryCollectionAsArray(ctx.collectionName, [], undefined, authOwner)).resolves.toEqual(expect.toIncludeSameMembers(
-            [ 
-                ...expectedItems, 
-                data.pagingMetadata(ctx.items.length, ctx.items.length)
-            ])
-        )
+        expect(response.data).toEqual({ results: expect.toIncludeSameMembers(ctx.items) })
+    
+        await expect(data.queryCollectionAsArray(ctx.collectionName, [], undefined, authOwner)).resolves.toEqual({
+            items: expect.toIncludeSameMembers(ctx.items),
+            pagingMetadata: data.pagingMetadata(ctx.items.length, ctx.items.length)
+        })
     })
 
     testIfSupportedOperationsIncludes(supportedOperations, [ AtomicBulkInsert, PrimaryKey ])('insert api should fail if item already exists', async() => {
         await schema.givenCollection(ctx.collectionName, [ctx.column], authOwner)
         await data.givenItems([ ctx.items[1] ], ctx.collectionName, authAdmin)
```

Comment:
I don't say it's not right but It's a little bit confusing to read this test and the one after it.
I think it would be better to change the test to be:
1. given ctx.items (instead specific item)
2. try to insert random item (I don't care if it would be [1], but if we are choosing random everywhere maybe it would be nice to do it here as well)
3. show it gets 409, query and get ctx.items (instead expect expectedItems).